python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -267,22 +273,27 @@ async def publish_data(self, data):
             The data recieved from the ESS device.
         """
         if self.ess_server.connected:
-            await self.ess_server.write_json(data)
+            array = data["telemetry"]["sensor_telemetry"]
+            line = ""
+            for idx, value in enumerate(array):
+                line += f"C0{idx}={value},"
+            line = line[:-1]
+            await self.ess_server.write_str(line)
 
     async def serial_temperature_task(self):
         """Get incoming data and publish through the server."""
         # Read sensors
         # wait for sample_wait_time between readings
-
+        self.log.info("Starting Serial Temperature loop.")
         while True:
             try:
-                self.log.debug(self.data)
+                self.log.info(f"{self.data=}")
```

Comment:
Please do not log at info level in loops. This will exhaust memory and make the docker container fail at some point.